packages/chain-adapters/src/utxo/UTXOChainAdapter.ts

```diff
@@ -72,11 +77,11 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
 
       switch (data.network) {
         case 'mainnet':
-          return caip2.toCAIP2({ chain: ChainTypes.Bitcoin, network: NetworkTypes.MAINNET })
+          return caip2.toCAIP2({ chain: this.bip44Params.coinType, network: NetworkTypes.MAINNET })
```

Comment:
BIP44 `coinType` !== `chainType`; surprised this typechecked!
```suggestion
          return caip2.toCAIP2({ chain: this.chainType, network: NetworkTypes.MAINNET })
```